garrysmod/lua/vgui/ddialogbox.lua

```diff
@@ -0,0 +1,203 @@
+
+local PANEL = {}
+
+-- See: https://en.wikipedia.org/wiki/Dialog_box
+
+function PANEL:Init()
+	self:SetTitle( "Untitled Dialog Box" )
+	self:SetSize( 400, 200 )
+	self:MakePopup()
+	self:Center()
+	self:SetDrawOnTop( true )
+	
+	self.children = {}
```

Comment:
Is there even a point of having this table, since you are putting every child you create into it?

You can just use `Panel:GetChildren()`, or the built-in `Panel:Clear()` for what this variable is being used for.